### ISSUE TITLE
Source for unlabeled data

### ISSUE THREAD

**Issue by function2-llx** [BAAI-DCAI/SegVol#5]
Thank you for your great work! According to the "2.1 Dataset Exploration" section of the paper, 90K unlabled open-srouce CT images are used in this work:

> Additionally, in order to build a universal feature extractor for volumetric medical images, we collect 90k unlabeled open-source CTs for pre-training.

However, in the "4.1.1 Data Collection and Normalization" section, only the 25 labeled datasets are introduced, while the source for those 90K unlabeld CT images are not mentioned. Since the those data are publicly available, would you kindly provide the source for them? This will greatly benefit future following works. Thanks again!

**Comment by baifanxxx** [comment]
We are processing data (approximately several TB) and looking for a suitable way to download it. It will be released soon. Of course, you can use the pre-trained model we provide first.
